float32 k_im # 0 -- 10
float32 d_im # 0 -- 1 default : 0.1

---

bool success